apps/web/src/app/[locale]/(email)/unsubscribe/page.tsx

```diff
@@ -0,0 +1,81 @@
+'use client';
+
+import { zodResolver } from '@hookform/resolvers/zod';
+import { Button } from '@repo/ui/components/button';
+import { DialogFooter } from '@repo/ui/components/dialog';
+import {
+  Form,
+  FormControl,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+} from '@repo/ui/components/form';
+import { Input } from '@repo/ui/components/input';
+import { useToast } from '@repo/ui/components/use-toast';
+import { useForm } from 'react-hook-form';
+import { z } from 'zod';
+import { removeFromWaitlist } from './_actions';
+
+const formSchema = z.object({
+  email: z.string().email(),
+});
+
+export type FormSchema = z.infer<typeof formSchema>;
+export default function UnsubscribePage(props: { searchParams: { email?: string } }) {
+  const { toast } = useToast();
+
+  const form = useForm<FormSchema>({
+    resolver: zodResolver(formSchema),
+    defaultValues: {
+      email: props.searchParams.email ?? '',
+    },
+  });
```

Comment:
FFT re: privacy policy, `?email=` as a query param will show up in any logs for page hits. this means sending the user's email to a service like Sentry, for example, anywhere we log the full request url

Reply:
hmm is that an actual concern for sentry? whats the usual workaround, encryption?

Reply:
:shrug: maybe not a concern, just a note

Reply:
nah this is valid and now im scared lollll